### ISSUE TITLE
Certificates has an inconsistent payload when empty

### ISSUE THREAD

**Issue by nijikokun** [Mashape/kong#2462]
### Summary

The default payload when you fetch a list that is empty across all other models looks like:

```js
// 20170427131035
// http://localhost:8001/upstreams

{
  "data": [
    
  ],
  "total": 0
}
```

Certificates empty payload looks like:

```js
// 20170427130651
// http://localhost:8001/certificates

{
  
}
```

### Steps To Reproduce

1. Fetch `upstreams` endpoint when there is no data
2. Fetch `certificates` endpoint when there is no date
3. Compare

### Additional Details & Logs

- Version: 0.10.1
- OS: OSX

**Comment by p0pr0ck5** [comment]
Nice eyes @nijikokun. Only pending question is whether to fix this in 0.10.2 or 0.11 as it could be considered a "breaking" change.

**Comment by p0pr0ck5** [comment]
0.10.2 it is!
